### DISCUSSION TITLE
Packing .net core 6.0 into appimage

### DISCUSSION THREAD

**Discussion by amitza** [AppImage/AppImageKit#1286] [Q&A]
I would like the pack a .net core 6 app as an appimage.

I've been manage to do most of the work using Pupnet-Deploy, but I have one issue:

The application depends on the existence of several Linux binaries like:
* lsusb
* lshw
* lw

And I really can't feature out how to pack them with the application.

Thanks for the help

**Comment by probonopd** [comment]
One way to do it: Make an AppImage for each of the Linux binaries, then unpack the AppImages and combine their contents. `AppRun` needs to set `$PATH` so that the binaries from inside the AppDir/AppImage are used.

If you can share what you have so far we may be able to help you better.
